Regressed by https://github.com/unicode-org/icu/commit/c3fe7e09d844

fs.c:266:32: error: use of undeclared identifier 'FALSE'
                update_platform_safe_name(d, FALSE, idx);
                                             ^

--- src/libltfs/fs.c.orig	2020-03-31 23:56:12 UTC
+++ src/libltfs/fs.c
@@ -263,7 +263,7 @@ struct dentry * fs_allocate_dentry(struct dentry *pare
 		d->platform_safe_name = NULL;
 	} else if (name && !platform_safe_name) {
 		d->name.name = strdup(name);
-		update_platform_safe_name(d, FALSE, idx);
+		update_platform_safe_name(d, false, idx);
 		if (! d->name.name || ! d->platform_safe_name) {
 			ltfsmsg(LTFS_ERR, 10001E, "fs_allocate_dentry: name");
 			if (d->name.name)
